data-changes/location-codes.md

```diff
@@ -1,45 +1,55 @@
 # Location codes change
 
-This document provides details of a correction we have applied in 2021 to correctly represent location code of GeoNet strong motion (SM) stations.
-The change has been applied to data and metadata which have been corrected during 2021. 
+This document provides details of **Data/Metadata Legacy corrections** we have applied to correctly label the **location code** of **strong motion instruments** part of the GeoNet national network for periods of time between **2011 and 2020**.
+The change was applied to **data and metadata** from **27 seismic stations**  (in between June and October 2021) 
```

Comment:
Is this what you mean?

"The change was applied between June & October 2021, to both data and metadata from 27 seismic stations".

Reply:
change applied by the Betta :)